okapi/locale/ro_RO/LC_MESSAGES/ro_RO.po

```diff
@@ -0,0 +1,432 @@
+msgid "hidden by"
+msgstr "ascunsă de"
+
+#: okapi/services/caches/formatters/gpxfile.tpl.php:75
+#, php-format
+msgid "%d recommendation"
+msgid_plural "%d recommendations"
+msgstr[0] "o recomandare"
+msgstr[1] "%d recomandări"
+msgstr[2] "%d recomandări"
+
+#: okapi/services/caches/formatters/gpxfile.tpl.php:76
+#, php-format
+msgid "found %d time"
+msgid_plural "found %d times"
+msgstr[0] "găsită o singură dată"
+msgstr[1] "găsită de %d ori"
+msgstr[2] "găsită de %d ori"
+
+# Generic term for trackable items (like geocoins and travelbugs).
+# and geokrets?
+#: okapi/services/caches/formatters/gpxfile.tpl.php:79
+#, php-format
+msgid "%d trackable"
+msgid_plural "%d trackables"
+msgstr[0] "un obiect călător"
+msgstr[1] "%d obiecte călătoare"
+msgstr[2] "%d obiecte călătoare"
+
+#: okapi/services/caches/formatters/gpxfile.tpl.php:83
+msgid "Personal notes"
+msgstr "Note personale"
+
+#: okapi/services/caches/formatters/gpxfile.tpl.php:87
+msgid "Attributes"
+msgstr "Atribute"
+
+#: okapi/services/caches/formatters/gpxfile.tpl.php:91
+msgid "Trackables"
+msgstr "Obiecte călătoare"
+
+#: okapi/services/caches/formatters/gpxfile.tpl.php:101
+#: okapi/services/caches/formatters/gpxfile.tpl.php:117
+msgid "Images"
+msgstr "Imagini"
+
+#: okapi/services/caches/formatters/gpxfile.tpl.php:124
+msgid "Spoilers"
+msgstr "Imagini (indicii?)"
+
+#: okapi/services/caches/formatters/gpxfile.tpl.php:133
+msgid "Image descriptions"
+msgstr "Descriere imagine"
+
+#: okapi/services/caches/formatters/gpxfile.tpl.php:141
+msgid "The cache probably is located in the following protection areas:"
+msgstr "Geocutia probabil că se află undeva în următoarele arii protejate:"
+
+#: okapi/services/caches/geocaches.php:535
+msgid "Parts of this geocache listing may be outdated."
+msgstr ""
+"O parte din informaţiile despre această geocutie s-ar putea să fie vechi şi "
+"neactualizate."
+
+#: okapi/services/caches/geocaches.php:536
+msgid "See the log entries for more information."
+msgstr "Vezi însemnările pentru mai multe informaţii."
+
+# For additional waypoints. As in "Stage 1: Parking".
+#: okapi/services/caches/geocaches.php:1015
+msgid "Stage"
+msgstr "Etapă"
+
+#: okapi/services/caches/geocaches.php:1059
+msgid "User location"
+msgstr "Poziţia utilizatorului"
+
+#: okapi/services/caches/geocaches.php:1062
+#, php-format
+msgid "Your own custom coordinates for the %s geocache"
+msgstr "Coordonatele modificate de tine pentru geocutia %s"
+
+# Is this the attribute - Nature?
+#: okapi/services/caches/geocaches.php:1222
+msgid "National Park / Landscape"
+msgstr "Rezervaţie naturală"
+
+#: okapi/services/caches/geocaches.php:1396
+#, php-format
+msgid ""
+"This <a href='%s'>geocache</a> description comes from the <a href='%s'>%s</"
+"a> site."
+msgstr ""
+"Descrierea <a href='%s'>geocutiei</a> provine de pe site-ul <a href='%s'>%s</"
+"a>."
+
+# as of %s - what is this string replaced with?
+# I assumed a date or year so far.
```

Comment:
If you mean geocaches.php line 1408, then @following5's interpretation is incomplete. 
First %s = $owner['profile_url']
Second %s = $owner['username']
Third %s = $cache_url
Fourth %s = $site_name
Fifth %s = strftime('%x') 

 %x     The preferred date representation for the current locale without
              the time.
So date, including year, month, day.

Please clarify the original meaning for this date (also for better english maybe).
1. Does it refer to the date when the cache was pulished (makes sense to declare a copyright notice)?
2. Does it refer to a predetermined date ?
3. Does it refer to the current date ? _(reading the code, it seems to me that way)_

Variants 2 and 3 make less sense, logically, for a copyright notice, for the reason that it is unclear what copyright terms were applied since the publication until the specified date.


Now, what is the meaning of this time expression?